influxql/engine.go

```diff
@@ -496,11 +496,19 @@ func (m *MapReduceJob) processAggregate(c *Call, reduceFunc ReduceFunc, resultVa
 		}
 	}
 
+	firstInterval := m.interval
+	if !m.stmt.IsRawQuery {
```

Comment:
I think due to the way we calculate our interval for raw queries, this passes because math works out that way.  This is more about not doing unnecessary work than it is about testing the calculation for raw queries.  In other words, raw queries would work with this calculation in there, but it simply isn't needed to be done again.

This calc is only for aggregate queries where the first time bucket is not a full bucket, and we need to offset the interval.